services/src/main/java/org/keycloak/protocol/oid4vc/issuance/TimeClaimNormalizer.java

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright 2025 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.keycloak.protocol.oid4vc.issuance;
+
+import java.time.Instant;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.time.temporal.ChronoUnit;
+import java.util.Locale;
+
+import org.keycloak.models.KeycloakSession;
+import org.keycloak.models.RealmModel;
+import org.keycloak.utils.StringUtil;
+
+import org.jboss.logging.Logger;
+
+/**
+ * Utility to apply correlation-mitigation to time-related claims
+ * by either randomizing within a window or rounding to a unit.
+ * <p>
+ * Configuration via realm attributes (all optional):
+ * - oid4vci.time.claims.strategy: off | randomize | round (default: off)
+ * - oid4vci.time.randomize.window.seconds: integer seconds (default: 86400)
+ * - oid4vci.time.round.unit: SECOND | MINUTE | HOUR | DAY (default: SECOND)
+ *
+ * @author <a href="mailto:Rodrick.Awambeng@adorsys.com">Rodrick Awambeng</a>
+ */
+public class TimeClaimNormalizer {
+
+    private static final Logger logger = Logger.getLogger(TimeClaimNormalizer.class);
+
+    public static final String ATTR_STRATEGY = "oid4vci.time.claims.strategy";
```

Comment:
Is it possible to move those constants to the `Oid4VciConstants` to ideally have single place with the various realm/client attributes related to OID4VCI?